File: Examples/test-suite/javascript/template_rename_runme.js
var template_rename = require("template_rename");

var i = new template_rename.iFoo();
var d = new template_rename.dFoo();

var a = i.blah_test(4);
var b = i.spam_test(5);
var c = i.groki_test(6);

var x = d.blah_test(7);
var y = d.spam(8);
var z = d.grok_test(9);

File: Examples/test-suite/lua/template_rename_runme.lua
require("template_rename")

local i = template_rename.iFoo()
local d = template_rename.dFoo()

local a = i:blah_test(4)
local b = i:spam_test(5)
local c = i:groki_test(6)

local x = d:blah_test(7)
local y = d:spam(8)
local z = d:grok_test(9)
